cloud-controller-manager/do/loadbalancers.go

```diff
@@ -275,27 +306,91 @@ func (l *loadBalancers) EnsureLoadBalancer(ctx context.Context, clusterName stri
 	}, nil
 }
 
-// UpdateLoadBalancer updates the load balancer for service to balance across
-// the droplets in nodes.
-//
-// UpdateLoadBalancer will not modify service or nodes.
-func (l *loadBalancers) UpdateLoadBalancer(ctx context.Context, clusterName string, service *v1.Service, nodes []*v1.Node) error {
+func getCertificateIDFromLB(lb *godo.LoadBalancer) string {
+	var id string
+	for _, rule := range lb.ForwardingRules {
+		if rule.CertificateID != "" {
+			id = rule.CertificateID
+			// CCM does not currently support multiple certificates on the
+			// loadbalancers it manages so we only need to grab the first one we find
+			break
```

Comment:
refactor: we do not need the `id` variable: this line can return `rule.CertificateID` directly, and 319 can return an empty string.